version.bzl

```diff
@@ -0,0 +1,22 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Version for the rules_nodejs repository
+"""
+
+# This version is synced with the version in package.json.
+# It will be automatically synced via the npm "version" script
+# that is run when running `npm version` during the release
```

Comment:
this comment is saying that there's some release machinery that needs to be changed so that this value continues to get updated. (scripts in the root package.json)
But it would be easier to leave this value in `index.bzl` - why move it? Is it a refactoring that's not critical to the feature you're adding? It will be easier to merge your PRs if you keep them minimal.

Reply:
There is machinery that updates this, it's the `on-version.js` script.  
we cant leave this in `index.bzl` since that would introduce a circular dep between `index.bzl` and `internal/npm_install/npm_install.bzl` - therefore it is critical for this feature.

Reply:
there is another place to update - in the package.json we have to stage the changes made by on-version so they're included in the release commit. I'll fix it